homeassistant/components/home_connect/sensor.py

```diff
@@ -80,10 +80,7 @@ async def async_update(self) -> None:
             else:
                 self._state = status[self._key].get(ATTR_VALUE)
                 if self._key == BSH_OPERATION_STATE:
-                    # Value comes back as an enum, we only really care about the
-                    # last part, so split it off
-                    # https://developer.home-connect.com/docs/status/operation_state
-                    self._state = self._state.split(".")[-1]
+                    self._state = OPERATION_STATE_TRANSLATION[self._state]
```

Comment:
this will crash if they ever decide to add a new operation state. i think you should have some fallback.

Reply:
Fixed it! Thanks!